src/test/java/AddRemoveElements.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.*;
+
+
+import static org.testng.Assert.assertEquals;
+
+public class AddRemoveElements extends BaseTest {
+
+
+    @BeforeMethod
+    public void navigate() {
+        super.setUp();
```

Comment:
It's better to add @before/after annotations to setuo() and exit() methods in BaseTest class so you won't need to call them each time in each nested class via super.setUp() but they will be executed automatically